src/Inventory.zig

```diff
@@ -174,6 +174,24 @@ pub const Sync = struct { // MARK: Sync
 				commands.pushBack(cmd);
 			}
 		}
+
+		fn setSpawn(newSpawnPoint: Vec3d) void {
```

Comment:
I don't think we should synchronize the spawn point with the client. I think the client shouldn't even know the spawn point. Instead I'd prefer if the client gets its respawn point sent by the server together with the kill packet.